README.md

```diff
@@ -13,10 +13,10 @@ the only required composer package to set up your project. To iniated a new
 project you can execute the following command:
 
 ```bash
-composer create-project ec-europa/subsite project-folder-name dev-master --no-interaction
+composer create-project ec-europa/subsite project-folder-name dev-release/4.x --no-interaction
```

Comment:
Use this example https://github.com/ec-europa/digit-qa-reference/blob/master/README.md as starting point to a good README.md